src/formatters/junit.js

```diff
@@ -0,0 +1,42 @@
+/*eslint no-console: "off"*/
+
+function printResults(results) {
+  const testCases = results.map(result => {
+    const file = result.filePath;
```

Comment:
If you named this const `name`, then you could use short-hand object notification below:

```
 _attributes: { name },
```

Reply:
Actually, it would be cleaner to remove the const altogether so that you can return immediately.

Reply:
As discussed, this cant be done without changing babel config